src/applications/gi/tests/e2e/00-required.cypress.spec.js

```diff
@@ -0,0 +1,69 @@
+import { initMockProfile, expectLocation } from './cypress-helpers';
+import institutionProfile from '../data/institution-profile.json';
+import searchResults from '../data/search-results.json';
+import { createId } from '../../utils/helpers';
+import {
+  typeOfInstitution,
+  search,
+  selectSearchResult,
+  displayLearnMoreModal,
+  collapseExpandAccordion,
+} from './gi-helpers';
+
+const institutionAttributes = institutionProfile.data.attributes;
+
+describe('Institution', () => {
+  beforeEach(() => {
+    cy.route('/v0/gi/institutions/', searchResults);
+
+    initMockProfile(institutionProfile);
```

Comment:
```suggestion
    initApplicationMock();
```
This method just sets the default mocks

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/tests/e2e/cypress-helpers.js#L35